browser/playlist/android/playlist_android_page_handler.cc

```diff
@@ -0,0 +1,25 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/playlist/android/playlist_android_page_handler.h"
+
+#include "brave/browser/playlist/playlist_service_factory.h"
+#include "chrome/browser/profiles/profile.h"
+
+PlaylistAndroidPageHandler::PlaylistAndroidPageHandler(Profile* profile)
+    : PlaylistPageHandler(profile) {
+  // DCHECK(playlistPageHandler_);
+  // TODO(deeppandya) : check if we need observer for android
```

Comment:
does this need to be done before merge? Is there an open issue for it?

Reply:
i don't think we need it. just confirm the change. i am updating right away.